lighthouse-core/lib/network-request.js

```diff
@@ -43,6 +43,9 @@ const RESOURCE_TYPES = {
   WebSocket: 'WebSocket',
   Other: 'Other',
   Manifest: 'Manifest',
+  SignedExchange: 'SignedExchange',
+  Ping: 'Ping',
+  CSPViolationReport: 'CSPViolationReport',
```

Comment:
nice!

while i do see these here https://chromedevtools.github.io/devtools-protocol/tot/Page#type-ResourceType

i don't see them here: https://github.com/ChromeDevTools/devtools-frontend/blob/master/front_end/common/ResourceType.js

but i guess that's because we're now ahead of the frontend. :)

Reply:
> but i guess that's because we're now ahead of the frontend. :)

🎉 💣 💥 💯 💃 🕺 🥇 

music to my ears!!!